### ISSUE TITLE
ML system setup

### ISSUE THREAD

**Issue by ksuderman** [lappsgrid-incubator/galaxy-paper-rank#9]
Setup the Python *learning system* environment to be used.  Check with @nancyide to determine what, exactly, is meant by *learning system*.
- Word2Vec
- BERT
- Something else?

**Tasks**
- [ ] Training: given a directory of text files learn *something*.
- [ ] Evalutation: given a text file return a yes/no answer

**Comment by nancyide** [comment]
My present thought is to go directly to using BERT (machine learning). I’d like Nadim to focus on setting up what is needed (once we have our corpus); there is a nice tutorial at https://towardsml.com/2019/09/17/bert-explained-a-complete-guide-with-theory-and-tutorial/

We should use the BERT pre-trained model as a base (and probably run on our test set just to see) and then add a layer trained on our data.

@keith_suderman: we will need considerable resources to run BERT. Should Nadim use jetstream?

Given how easy it is to generate embeddings using word2vec, we probably should still generate them, and do it for the positives and the negatives separately. However, this is an afterthought if we are using BERT, and so it is not necessary to do now. I have Python scripts that can do it.
